docs/how-to-guides/themes/block-theme-overview.md

```diff
@@ -119,6 +126,8 @@ As we're still early in the process, the number of blocks specifically dedicated
 -   Post Featured Image
 -   Post Hierarchical Terms
```

Comment:
Renamed Post Categories

Both post categories and post tags are variations of the post term block -Which block is correct to mention here?

Reply:
I think it's better to mention what's actually available in the editor, so Post Categories would be our best choice

Reply:
Fixed in the last commit